src/overview/actions.js

```diff
@@ -64,5 +65,38 @@ export function refreshSearch({loadingIndicator = false}) {
     }
 }
 
+// Getting more results
+export function getMoreResults({loadingIndicator = false, endDate, scrollPosition}) {
+    return async function (dispatch, getState) {
+        const { query, startDate } = ourState(getState())
+
+        if (loadingIndicator) {
+            // Show to the user that search is busy
+            dispatch(showLoadingIndicator())
```

Comment:
why "getMoreResult" knows about what to "show"? This seems a mix of business logic and view logic.